### PR TITLE
added makefile and package.json

### PR DESCRIPTION
**Pull request by catevita**
this eases the effort to see if everything compiles

### CODE REVIEW

**Comment by catevita** [comment]
ptal @nnmm @feuerste

File: package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "workspaces": [
+        "octree_web_viewer/client",
+        "xray/client"
+    ],
+    "scripts": {
+        "build_all": "yarn install --silent --ignore-engines && yarn wsrun build",
+        "test": "yarn wsrun --exclude-missing test"
+    },
+    "devDependencies": {
+        "typescript": "^3.3.3",
```
Comment (feuerste) [inline package.json]: Shall we use the latest versions here, i.e. 3.5.1 for typescript and 3.6.6 for wsrun?

Comment (catevita) [reply package.json]: As I am not so proficient with typescript, I am not so sure, would it clash with other projects (e.g. avmg)?

**Comment by nnmm** [comment]
What is package.json needed for? `yarn build_all` works without that, it seems.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+DEFAULT_GOAL := all
```
Comment (nnmm) [inline Makefile]: Nit: The first goal is already the default

**Comment by nnmm** [comment]
Looks like I was wrong, yarn build_all doesn't work without the package.json

**Comment by catevita** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user catevita. Merge queue now has a length of 1.

**Pull request merged**